### DISCUSSION TITLE
Enhancement: Allow Temperetures Above 1

### DISCUSSION THREAD

**Discussion by Oddity-Official** [danny-avila/LibreChat#4463] [Feature Requests & Suggestions]
### What features would you like to see added?

Please allow us to set LLM temperature values above 1.
For more novice users, it would also be useful to have sections highlighted within the slider to show general "ideal" values. 

### More details

Temperatures above 1 allow us to explore more esoteric, and arguably more creative, outputs from LLMs. There's no realistic reason to constrain it to a max of 1. 

### Which components are impacted by your request?

General, UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
1 is just the maximum for OpenAI, and then as a result, custom endpoints. I plan on making it configurable.

**Reply by danny-avila** [reply]
I just reviewed the documentation again, and I will change it to 2 to match
